config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "prefix": "!",
-  "token": "setmeplease"
+  "prefix": "-",
+  "token": "NTI1MTMzMTc1MTEwNzYyNTA2.XmE0Qg.kByJJ3VOLyWmMQljldsc-j92aTk"
```

Comment:
Might not want your Bot Token public like this.

Reply:
Ok